libsolidity/ast/Types.cpp

```diff
@@ -406,28 +603,52 @@ bool IntegerConstantType::isImplicitlyConvertibleTo(Type const& _convertTo) cons
 			return true;
 		return false;
 	}
+	else if (_convertTo.category() == Category::FixedPoint)
+	{
+		auto targetType = dynamic_cast<FixedPointType const*>(&_convertTo);
```

Comment:
As discussed below, I think we should rather use `fixedPoint() && fixedPoint()->isImplicitlyConvertibleTo(_convertTo)` to avoid code duplication.